deepmd/pt/loss/ener.py

```diff
@@ -153,3 +160,46 @@ def forward(self, model_pred, label, natoms, learning_rate, mae=False):
         if not self.inference:
             more_loss["rmse"] = torch.sqrt(loss.detach())
         return loss, more_loss
+
+    @property
+    def label_requirement(self) -> List[DataRequirementItem]:
```

Comment:
I think it can be dynamic. For example, if no virial loss is set, the virial requirement don't need to be added.